src/hackney_ssl.erl

```diff
@@ -49,6 +49,13 @@ connect(Host, Port, Opts) ->
 
 connect(Host, Port, Opts, Timeout) when is_list(Host), is_integer(Port),
                                         (Timeout =:= infinity orelse is_integer(Timeout)) ->
+  Host1 = try
```

Comment:
would be better to use  `inet:parse_address/1` there. Can you update your PR to use it? (sorry for the late answer)

Reply:
Sure, no problem. Just to be sure, you mean something like this, right?
```
f(Host) ->
    Host1 = case inet:parse_address(Host) of
             {ok, Address} -> Address;
             {error, _} -> Host
         end,
    Host1.
```
Would it be better if I moved this to a separate function, or should it remain inline in `connect/4`?

I will probably update the PR when I get home in the evening (can't really code right now, family&stuff...)

Reply:
maybe in a separate function to make it more readable.

Reply:
@piotrklibert would be cool if you can send your commit today to be part of the release. Let me know :)